### PR TITLE
COMMERCE-11553 - Product with variants shows Unavailable label even though it has enough stock

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/COMMERCE-11553
Resent from https://github.com/liferay-commerce/liferay-portal/pull/3828

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 1]
```diff
@@ -1460,6 +1460,113 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11553. If a product with variants follows a product without stock, only the product without stock shows the unavailable label"
+	@priority = 5
+	test CanViewProductWithStockAvailableWhenFollowingAZeroStockProduct {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:1465]: I think this name may be better 

test CanViewProductWithStockAvailableWhenItFollowsAProductWithoutStock

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 2]
```diff
@@ -1460,6 +1460,113 @@ definition {
...
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And an SKU Contributor option with two values is set to the 'Ball Joints' product") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:1486]: "And a SKU Contributor ... is assigned to the 'Ball Joints' product"

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
@victoroscuro please force-forward this pr since no related errors to this pull, thanks.

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**